instrumentation/camel-2.20/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/AwsSpanAssertions.java

```diff
@@ -0,0 +1,187 @@
+
+class AwsSpanAssertions {
+
+  private AwsSpanAssertions() {}
+
+  static void sqs(SpanDataAssert span, int index, String spanName) {
+    sqs(span, index, spanName, null, null, CLIENT, null);
+  }
+
+  static void sqs(SpanDataAssert span, int index, String spanName, String queueUrl) {
+    sqs(span, index, spanName, queueUrl, null, CLIENT, null);
+  }
+
+  static void sqs(
+      SpanDataAssert span, int index, String spanName, String queueUrl, String queueName) {
+    sqs(span, index, spanName, queueUrl, queueName, CLIENT, null);
+  }
+
+  public static void sqs(
+      SpanDataAssert span,
+      int index,
+      String spanName,
+      String queueUrl,
+      String queueName,
+      SpanKind spanKind) {
+    sqs(span, index, spanName, queueUrl, queueName, spanKind, null);
+  }
+
+  static void sqs(
+      SpanDataAssert span,
+      int index,
+      String spanName,
+      String queueUrl,
+      String queueName,
+      SpanKind spanKind,
+      SpanData parentSpan) {
+    if (index == 0) {
+      span.hasNoParent();
+    } else {
+      span.hasParent(parentSpan);
+    }
+
+    span.hasName(spanName)
+        .hasKind(spanKind)
+        .hasAttributesSatisfyingExactly(
+            equalTo(stringKey("aws.agent"), "java-aws-sdk"),
+            satisfies(stringKey("aws.endpoint"), val -> val.isInstanceOf(String.class)),
+            equalTo(SemanticAttributes.HTTP_STATUS_CODE, 200),
+            satisfies(
+                stringKey("aws.queue.name"),
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isEqualTo(queueName), v -> assertThat(v).isNull())),
+            satisfies(
+                stringKey("aws.queue.url"),
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isEqualTo(queueUrl), v -> assertThat(v).isNull())),
+            equalTo(SemanticAttributes.HTTP_METHOD, "POST"),
+            satisfies(
+                SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH,
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isNull(), v -> assertThat(v).isInstanceOf(Long.class))),
+            satisfies(
+                SemanticAttributes.HTTP_REQUEST_CONTENT_LENGTH,
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isNull(), v -> assertThat(v).isInstanceOf(Long.class))),
+            satisfies(
+                SemanticAttributes.USER_AGENT_ORIGINAL,
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isNull(),
+                        v -> assertThat(v).isInstanceOf(String.class))),
+            satisfies(
+                SemanticAttributes.HTTP_URL,
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isNull(),
+                        v -> assertThat(v).isInstanceOf(String.class))),
```

Comment:
Originally this did not have the `.isNull()` assertion
```suggestion
            satisfies(
                SemanticAttributes.HTTP_URL,
                val -> val.isInstanceOf(String.class)),
```